openedx/core/djangoapps/content/course_overviews/management/commands/backfill_history.py

```diff
@@ -0,0 +1,138 @@
+"""
+Management command to backfille history.
+"""
+import logging
+import os
+import time
+from django.core.management.base import BaseCommand
+from django.db import connection, transaction
+
+log = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    Backfille history for models using django-simple-history.
+    Example usage:
+    $ ./manage.py lms backfill_history --batchsize 1000 --sleep_between 1 --settings=devstack
+    """
+
+    help = (
+        "Populates the historical records with an initial state as of 06/29"
+    )
+
+    # Size in bytes to read from input file per batch
+    DEFAULT_SIZE = 1000000
+    DEFAULT_SLEEP_BETWEEN_INSERTS = 1
+    DATE = '2019-06-29'
+    HISTORY_USER_ID = None
+    HISTORY_CHANGE_REASON = 'initial history population'
+
+    TABLES = [
+        #{'name': 'organizations_organization', 'exclude_column': None, 'input_filename': 'organizations_organization_2019_06_29.csv'},
+        {'name': 'entitlements_courseentitlement', 'exclude_column': None, 'input_filename': 'entitlements_courseentitlement_2019_06_29.csv'}
+    ]
+
+    def add_arguments(self, parser):
+        super(Command, self).add_arguments(parser)
+
+        parser.add_argument(
+            '--sleep_between',
+            default=self.DEFAULT_SLEEP_BETWEEN_INSERTS,
+            type=float,
+            help='Seconds to sleep between chunked inserts.'
+        )
+
+        parser.add_argument(
+            "--size",
+            action="store",
+            default=self.DEFAULT_SIZE,
+            type=int,
+            help="Maximum number of bytes to read from input file in each batch.",
+        )
+
+        parser.add_argument(
+            "--input_root",
+            action="store",
+        )
```

Comment:
I've included the help. Not sure what I should default it to, this would be the local directory on the box from where the management would read the input files.